vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/overview/components/ProjectOverviewComponent.java

```diff
@@ -285,12 +241,30 @@ private void setUpResponsiblePersonSearch() {
       setUpPersonSearch(projectCreationDialog.responsiblePerson);
     }
 
-    private void preloadContentFromOffer(String offerId) {
-      log.info("Receiving offerId " + offerId);
-      OfferId id = OfferId.from(offerId);
-      Optional<Offer> offer = offerLookupService.findOfferById(id);
-      offer.ifPresentOrElse(projectCreationDialog::setOffer,
-          () -> log.error("No offer found with id: " + offerId));
+    private void setUpProjectManagerSearch() {
+      setUpPersonSearch(projectCreationDialog.projectManager);
+    }
+
+    private void processProjectCreation(ProjectCreationContent projectCreationContent) {
+
+      Result<Project, ApplicationException> project = projectRegistrationService.registerProject(
+          projectCreationContent.offerId(), projectCreationContent.projectCode(),
+          projectCreationContent.title(), projectCreationContent.objective(),
+          projectCreationContent.experimentalDesignDescription(), projectCreationContent.species(),
+          projectCreationContent.specimen(), projectCreationContent.analyte(),
+          projectCreationContent.principalInvestigator(),
+          projectCreationContent.projectResponsible(),
+          projectCreationContent.projectManager());
+
+      project
+          .onValue(result -> {
+            displaySuccessfulProjectCreationNotification();
+            projectCreationDialog.resetAndClose();
+            projectGrid.getDataProvider().refreshAll();
+          })
+          .onError(e -> {
+            throw e;
+          });
```

Comment:
Do we want to throw errors on the frontend?
 I'd argue we could implement and call a method such as "displayProjectCreationFailure()"  similiar to displaySuccessfulProjectCreation() which shows an error notification to the user

Reply:
This is handled globally